composer.json

```diff
@@ -19,7 +19,7 @@
 
     "require": {
         "php":                      ">=5.3.3",
-        "phpspec/prophecy":         "~1.1",
+        "phpspec/prophecy":         "dev-master",
```

Comment:
@everzet we need a proper release tag to depend on. We are waiting for your next release.